docs/CaliptraHardwareSpecification.md

```diff
@@ -18,6 +19,19 @@ This document provides definitions and requirements for a Caliptra cryptographic
 
 For information on the Caliptra Core, see the [High level architecture](https://chipsalliance.github.io/Caliptra/doc/Caliptra.html#high-level-architecture) section of [Caliptra: A Datacenter System on a Chip (SoC) Root of Trust (RoT)](https://chipsalliance.github.io/Caliptra/doc/Caliptra.html).
 
+## Key Caliptra Core 2.0 Changes
+* AXI subordinate replaces APB interface of Caliptra 1.x hardware
+* SHA Accelerator functionality now available exclusively over mailbox
+    * SHA Accelerator adds new SHA save/restore functionality
+* Adams Bridge Dilithium/ML-DSA (refer to [Adams bridge spec](https://github.com/chipsalliance/adams-bridge/blob/main/docs/Adams\ bridge_HardwareSpecs.docx))
```

Comment:
This one is also a typo from an older suggestion.... instead of "Adams\ bridge_HardwareSpecs..." it should be "Adams%20bridge_HardwareSpecs".
Broken link.